Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcessInputTest.py

```diff
@@ -0,0 +1,96 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import unittest
+from mantid.simpleapi import SANSILLAutoProcess, config, mtd
+
+
+class SANSILLAutoProcessInputTest(unittest.TestCase):
+
+    def setUp(self):
+        config.appendDataSearchSubDir("ILL/D11/")
+        self._facility = config["default.facility"]
+        self._instrument = config["default.instrument"]
+        config["default.facility"] = "ILL"
+        config["default.instrument"] = "D11"
+
+    def tearDown(self):
+        config["default.facility"] = self._facility
+        config["default.instrument"] = self._instrument
+        mtd.clear()
+
+    def test_validateInput(self):
```

Comment:
Can you split this into individual tests with representative names?
Like `test_wrong_absorber_dim` , etc.